features/features.go

```diff
@@ -46,6 +48,7 @@ var features = map[FeatureFlag]bool{
 	AllowRenewalFirstRL:         false,
 	WildcardDomains:             false,
 	EnforceChallengeDisable:     false, // deprecated
+	RPCHeadroom:                 true,
```

Comment:
I think using `true` here results in `config/` and `config-next/` being the same w.r.t the `RPCHeadroom` feature flag.